toolchains/ios.toolchain.cmake

```diff
@@ -20,6 +20,8 @@
 
 include( "${CMAKE_CURRENT_LIST_DIR}/apple.cmake" )
 
+add_compile_options( $<$<COMPILE_LANGUAGE:CXX>:-x objective-c++> )
```

Comment:
this is a hack...which is sometimes necessary but then one should do it for a specific file in his own build scripts...this cannot go in as a default....that's what the .mm file extension is for...